frontend/packages/local-storage-operator-plugin/locales/zh/lso-plugin.json

```
@@ -30,7 +30,7 @@
   "Selecting all nodes will use the available disks that match the selected filters only on selected nodes.": "选择所有节点将使用在所选节点上与所选过滤器相匹配的可用磁盘。",
   "Disk Type": "磁盘类型",
   "Advanced": "高级",
-  "Disk Mode": "磁盘模式",
+  "Volume Mode": "音量模式",
```

```suggestion
"Disk Mode": "音量模式",
```


We dont need to change in language-specific localization file